generate/write-to-json-file.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  const allFileContent = arrayNewItems.map(item => {
+    return applyFilter ? filterProperties(config.propertiesToFilter, item) : item
+  })
+  writeToFile(DEST_JSON_FILE, stringify(allFileContent))
+}
+
+export function writeJsonObject(newItem, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  const itemToCopy = applyFilter ? filterProperties(config.propertiesToFilter, newItem) : newItem
+  writeToFile(DEST_JSON_FILE, stringify(itemToCopy))
```

Comment:
Could this just be:

```javascript
writeToFile(DEST_JSON_FILE, JSON.stringify(itemToCopy))
```

I don't think we need to do any spacing of any kind here...